### PR TITLE
Update Warmup LR pic 

### PR DESCRIPTION
**Pull request by TNTWEN**
支持warmup曲线的绘制   warmup每个batch 学习率都在上升，在绘图时取的是每个epoch最后的学习率

### CODE REVIEW

**Comment by Huangdebo** [comment]
> 支持warmup曲线的绘制 warmup每个batch 学习率都在上升，在绘图时取的是每个epoch最后的学习率

如果过warmup 大于1个 epoch，源码是可以画出来的（绘图时取的是每个epoch最后的学习率），只是不能画出warmup过程中每个batch中 lr 的上升。修改一下代码就可以画出来了，就像 readme 中的第四张图

**Comment by TNTWEN** [comment]
如果按batch来画就像这样
image

主要是在设置warmup后plot_lr_scheduler函数绘制出来的图怪怪的，哈哈

**Comment by TNTWEN** [comment]
@Huangdebo 
我重新更新了一下，这次还是用plot_lr_scheduler 函数
现在可以画出warmup的图像了，并且支持选择epoch或者iteration作为横轴 作图
resume 应该也没有问题
您看看还有没有bug

**Comment by TNTWEN** [comment]
epoch作为横轴
image
image

iteration作为横轴时候，不太适合用点横线了，就用直线
image
image

**Comment by Huangdebo** [comment]
> epoch作为横轴 image image
> 
> iteration作为横轴时候，不太适合用点横线了，就用直线 image image

感谢这么关注，最近实在太忙了，过些时间我再详细看看，然后把修复的部分融合进来
